client/components/data/query-site-keyrings/index.js

```diff
@@ -0,0 +1,56 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import { isRequestingSiteKeyrings } from 'state/site-keyrings/selectors';
+import { requestSiteKeyrings } from 'state/site-keyrings/actions';
+
+class QuerySiteKeyrings extends Component {
+	componentWillMount() {
+		this.requestKeyrings( this.props );
```

Comment:
The request [should be](https://stackoverflow.com/questions/27139366/why-do-the-react-docs-recommend-doing-ajax-in-componentdidmount-not-componentwi?utm_medium=organic&utm_source=google_rich_qa&utm_campaign=google_rich_qa) on `componentDidMount`